airflow/cli/cli_parser.py

```diff
@@ -540,9 +541,18 @@ def string_lower_type(val):
 )
 ARG_IGNORE_DEPENDS_ON_PAST = Arg(
     ("-I", "--ignore-depends-on-past"),
-    help="Ignore depends_on_past dependencies (but respect upstream dependencies)",
+    help="Deprecated -- use `--depends-on-past ignore` instead. "
+    "Ignore depends_on_past dependencies (but respect upstream dependencies)",
     action="store_true",
 )
+ARG_DEPENDS_ON_PAST = Arg(
+    ("-d", "--depends-on-past"),
+    help="Determine how Airflow should deal with past dependencies. The default action is `check`, Airflow "
+    "will check if the the past dependencies are met for the tasks having `depends_on_past=True` before run "
+    "them, if `ignore` is provided, the past dependencies will be ignored.",
+    type=PastDependenciesAction,
+    default=PastDependenciesAction.CHECK,
+)
```

Comment:
I’d be nice to add a `choices` key so argparse can emit better help message.

Reply:
I had to change the argument type to `str` because argparse loads the input argument as an an object of the Enum before comparing with the choices, and If I provide the Enum objects list as choices, the help message will be very ugly.

Reply:
Hmm, if we’re only doign string comparison, I wonder if it’s better to simply use `Literal` for this. the value is always coming from argparse, so we don’t really need the enum for validation.

Reply:
I agree, done!